.build/images/Quartz64/quartz64_defconfig

```diff
@@ -1122,3 +1134,16 @@ CONFIG_ZSWAP=y
 CONFIG_ZSWAP_COMPRESSOR_DEFAULT_ZSTD=y
 CONFIG_ZSWAP_DEFAULT_ON=y
 CONFIG_ZSWAP_ZPOOL_DEFAULT_ZSMALLOC=y
+CONFIG_RT_GROUP_SCHED=y
+CONFIG_INET_ESP=m
+CONFIG_INET_ESPINTCP=n
+CONFIG_INET_ESP_OFFLOAD=n
+CONFIG_XFRM_USER=m
+CONFIG_XFRM=y
+CONFIG_XFRM_ALGO=m
+CONFIG_XFRM_ESP=m
+CONFIG_XFRM_INTERFACE=n
+CONFIG_XFRM_MIGRATE=n
+CONFIG_XFRM_STATISTICS=n
+CONFIG_XFRM_SUB_POLICY=n
```

Comment:
Same here, why is it needed to explicitly disable those features? Or is it just that they get implicitly enabled with `CONFIG_XFRM=y`, and your aim was to enable only the minimal additional feature set?